seattle_lidar/anaconda-project.yml

```diff
@@ -0,0 +1,54 @@
+# To reproduce: install 'anaconda-project', then 'anaconda-project run'
+name: seattle_lidar
+description: Visualize Lidar Scattered Point Elevation Data in Seattle
+maintainers:
+  - jbednar
+labels:
+  - datashader
+  - geoviews
+
+user_fields: [labels, skip, maintainers]
+
+channels: [conda-forge]
+
+packages: &pkgs
+  - python=3.6
+  - notebook==6.0.3
+  - nomkl
+  - cartopy==0.17.0
+  - datashader==0.10.0
+  - fastparquet==0.3.3
+  - geoviews==1.6.6
+  - holoviews==1.12.7
+  - python-snappy=0.5.4
+  - intake==0.5.4
+  - intake-xarray==0.3.1
+  - rasterio==1.0.25
+  - dask==2.9.2
+  - s3fs==0.4.0
+  - pandas==0.25.3
+  - distributed=2.9.3
```

Comment:
What's the difference between = and ==? Presumably it should pick one or the other, preferably for all of the examples on this site? Same for spacing; is the whitespace in " <6"  significant?

Reply:
Fixed in f9e7230 - I agree the formatting should be consistent.